Alignment/OfflineValidation/bin/DMRtrends.cc

```diff
@@ -0,0 +1,749 @@
+      histo->SetBinError(histo->FindBin(x[i]),pow(10,-6));
+    }
+  return histo;
+}
+
+
+/*! \fn PlotDMRTrends
+ *  \brief Plot the DMR trends.
+ */
+
+void PlotDMRTrends(vector<string> labels, TString Year, string myValidation, vector<string> geometries, vector<Color_t> colours, TString outputdir, bool pixelupdate, vector<int> pixelupdateruns, bool showlumi){
+    gErrorIgnoreLevel = kWarning;
+    checkrunlist(pixelupdateruns,{},Year);
+    vector<TString> structures { "BPIX", "BPIX_y", "FPIX", "FPIX_y", "TIB", "TID", "TOB", "TEC"};
+
+    const map<TString,int> nlayers{ {"BPIX", 4}, {"FPIX", 3}, {"TIB", 4}, {"TID", 3}, {"TOB", 6}, {"TEC", 9} };
+
+
+    TString filename=myValidation+"DMRtrends_"+labels.at(1)+"_"+labels.at(0)+".root";
+    TFile *in= new TFile(filename);
+    for (TString& structure: structures) {
+        TString structname = structure;
+        structname.ReplaceAll("_y", "");
+        int layersnumber=nlayers.at(structname);
+        for (int layer=0; layer<=layersnumber;layer++){
+            vector<TString> variables {"mu", "sigma", "muplus", "sigmaplus", "muminus", "sigmaminus", "deltamu", "sigmadeltamu", "musigma", "muplussigmaplus", "muminussigmaminus", "deltamusigmadeltamu"};
+            vector<string> YaxisNames { "#mu [#mum]", "#sigma_{#mu} [#mum]", "#mu outward [#mum]", "#sigma_{#mu outward} [#mum]", "#mu inward [#mum]", "#sigma_{#mu inward} [#mum]", "#Delta#mu [#mum]", "#sigma_{#Delta#mu} [#mum]", "#mu [#mum]", "#mu outward [#mum]", "#mu inward [#mum]", "#Delta#mu [#mum]",}; 
+	    //For debugging purposes we still might want to have a look at plots for a variable without errors, once ready for the PR those variables will be removed and the iterator will start from 0
+            for(size_t i=8; i < variables.size(); i++){
+                TString variable= variables.at(i);
+                TCanvas * c = new TCanvas("dummy","",2000,800);
+                vector<Color_t>::iterator colour = colours.begin();
+
+                TMultiGraph *mg = new TMultiGraph(structure,structure);
+                THStack *mh = new THStack(structure,structure);
+		size_t igeom=0;
+                for (string geometry: geometries) {
+                    TString name = getName(structure, layer, geometry);
+                    TGraphErrors *g = dynamic_cast<TGraphErrors*> (in->Get(name+"_"+variables.at(i)));
+		    g->SetName(name+"_"+variables.at(i));
+                    if(i>=8){
+                        g->SetLineWidth(1);
+			g->SetLineColor(*colour);
+                        g->SetFillColorAlpha(*colour,0.2);
+                    }
+		    vector<vector<double>> vectors; 
+		    //if(showlumi&&i<8)scalebylumi(dynamic_cast<TGraph*>(g));
+		    if(showlumi)scalebylumi(g,Year);
+		    g->SetLineColor(*colour);
+                    g->SetMarkerColor(*colour);
+		    TH1F *h = ConvertToHist(g); 
+ 		    h->SetLineColor(*colour);
+ 		    h->SetMarkerColor(*colour);
+ 		    h->SetMarkerSize(0);
+		    h->SetLineWidth(1.5);
```

Comment:
@cardinia, this line seems to be the culprit of the clang compilation warning:
```
Alignment/OfflineValidation/bin/DMRtrends.cc
/build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_10_6_X_2019-03-18-2300/src/Alignment/OfflineValidation/bin/DMRtrends.cc:598:23: warning: implicit conversion from 'double' to 'Width_t' (aka 'short') changes value from 1.5 to 1 [-Wliteral-conversion]
```
can you fix it?

Reply:
Yes, it's an easy fix. Unfortunately I got sick, but I'll fix it once I'm back to work.